core/templates/components/state-directives/hint-editor/hint-editor.component.html

```diff
@@ -1,56 +1,60 @@
 <div class="h-100">
-  <div ng-if="!$ctrl.hintEditorIsOpen"
+  <div *ngIf="!hintEditorIsOpen"
        class="h-100"
-       ng-attr-title="<[$ctrl.isEditable ? 'Edit hint' : '']>">
+       [attr.title]="isEditable ? 'Edit hint' : ''">
     <div class="oppia-readonly-rule-tile"
-         ng-class="{'oppia-editable-section': $ctrl.isEditable}">
+         ngClass="{'oppia-editable-section': isEditable}">
       <div class="oppia-rule-preview oppia-transition-200">
-        <div class="oppia-click-to-start-editing protractor-test-open-hint-editor"
-             ng-if="$ctrl.isEditable" ng-click="$ctrl.openHintEditor()">
-          <i ng-if="$ctrl.isEditable"
+        <div class="oppia-click-to-start-editing  protractor-test-open-hint-editor"
+             *ngIf="isEditable" (click)="openHintEditor()">
+          <i *ngIf="isEditable"
              class="material-icons oppia-editor-edit-icon float-right"
              title="Edit Hint">&#xE254;
           </i>
         </div>
 
-        <strong>Hint #<[$ctrl.getIndexPlusOne()]> is...</strong>
+        <strong>Hint #{{indexPlusOne}} is...</strong>
         <span>
-          <angular-html-bind class="oppia-rte-editor" html-data="$ctrl.hint.hintContent.html">
-          </angular-html-bind>
+          <angular-html-bind-wrapper classStr="oppia-rte-editor"
+                                     [htmlData]="hint.hintContent.html">
+          </angular-html-bind-wrapper>
         </span>
       </div>
     </div>
   </div>
 
-  <div ng-if="$ctrl.isEditable && $ctrl.hintEditorIsOpen">
+  <div *ngIf="isEditable && hintEditorIsOpen">
     <form role="form"
           class="form-inline"
-          name="$ctrl.editHintForm"
-          ng-submit="$ctrl.saveThisHint()">
+          name="editHintForm"
+          [formGroup]="editHintForm"
+          ngSubmit="saveThisHint()">
```

Comment:
Should this be `(ngSubmit)`?

Reply:
Has this been tested?

Reply:
yes ` (ngSubmit) ` should be used, i don't know why this is working fine while testing with  `ngSubmit` now this is replaced with (ngSubmit).

Reply:
`Has this been tested?` after more investigation i came accrossed that we are not using form here form code is  useless so it is removed now PTAL

Reply:
`Proof**`


https://user-images.githubusercontent.com/77006897/148692148-bae7ff96-fa22-41ed-85f5-4fd5ffe9a2e8.mp4

Reply:
@kevintab95 PTAL that`s why error is not showed up earlier

Reply:
Thanks for fixing it!

Reply:
welcome